### ISSUE TITLE
Build of client fails with "Error: Unrecognized option 'enableSourcemap'"

### ISSUE THREAD

**Issue by gmacario** [aquariophilie/blobfishes#128]
Steps to reproduce

```bash
cd client
npm i
npm run build
```

Result:

```text
gmacario@gmpowerhorse:~/github/aquariophilie/blobfishes/client (main)*$ npm run build

> ugly-frontend@0.0.1 build /home/gmacario/github/aquariophilie/blobfishes/client
> svelte-kit build

vite v2.6.13 building for production...
transforming (67) node_modules/sveltestrap/src/ListGroup.svelte
warn - You have enabled the JIT engine which is currently in preview.
warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.
✓ 232 modules transformed.
.svelte-kit/output/client/_app/manifest.json                               2.04 KiB
.svelte-kit/output/client/_app/pages/__layout.svelte-09e983a2.js           2.85 KiB / gzip: 1.31 KiB
.svelte-kit/output/client/_app/error.svelte-ec339748.js                    1.56 KiB / gzip: 0.75 KiB
.svelte-kit/output/client/_app/pages/index.svelte-93192d1a.js              0.64 KiB / gzip: 0.39 KiB
.svelte-kit/output/client/_app/pages/about.svelte-a5899d97.js              0.58 KiB / gzip: 0.39 KiB
.svelte-kit/output/client/_app/start-4ea01552.js                           19.01 KiB / gzip: 6.93 KiB
.svelte-kit/output/client/_app/assets/start-464e9d0a.css                   0.19 KiB / gzip: 0.15 KiB
.svelte-kit/output/client/_app/pages/authors.svelte-9696c7a2.js            10.08 KiB / gzip: 3.53 KiB
.svelte-kit/output/client/_app/assets/vendor-cf063f61.css                  0.09 KiB / gzip: 0.08 KiB
.svelte-kit/output/client/_app/pages/books.svelte-4829577c.js              13.96 KiB / gzip: 4.55 KiB
.svelte-kit/output/client/_app/assets/pages/__layout.svelte-d545374f.css   8.83 KiB / gzip: 2.54 KiB
.svelte-kit/output/client/_app/chunks/vendor-16ce580a.js                   83.09 KiB / gzip: 23.24 KiB
vite v2.6.13 building SSR bundle for production...
✓ 4 modules transformed.
[vite-plugin-svelte] Unrecognized option 'enableSourcemap'
file: /home/gmacario/github/aquariophilie/blobfishes/client/.svelte-kit/build/components/error.svelte
> Unrecognized option 'enableSourcemap'
Error: Unrecognized option 'enableSourcemap'
    at file:///home/gmacario/github/aquariophilie/blobfishes/client/node_modules/svelte/compiler.mjs:30015:19
    at Array.forEach (<anonymous>)
    at validate_options (file:///home/gmacario/github/aquariophilie/blobfishes/client/node_modules/svelte/compiler.mjs:30009:26)
    at compile (file:///home/gmacario/github/aquariophilie/blobfishes/client/node_modules/svelte/compiler.mjs:30053:5)
    at compileSvelte2 (file:///home/gmacario/github/aquariophilie/blobfishes/client/node_modules/@sveltejs/vite-plugin-svelte/dist/index.js:313:20)
    at async Object.transform (file:///home/gmacario/github/aquariophilie/blobfishes/client/node_modules/@sveltejs/vite-plugin-svelte/dist/index.js:1460:27)
    at async ModuleLoader.addModuleSource (/home/gmacario/github/aquariophilie/blobfishes/client/node_modules/rollup/dist/shared/rollup.js:22148:30)
    at async ModuleLoader.fetchModule (/home/gmacario/github/aquariophilie/blobfishes/client/node_modules/rollup/dist/shared/rollup.js:22201:9)
    at async /home/gmacario/github/aquariophilie/blobfishes/client/node_modules/rollup/dist/shared/rollup.js:22176:48
    at async Promise.all (index 1)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ugly-frontend@0.0.1 build: `svelte-kit build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ugly-frontend@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gmacario/.npm/_logs/2021-11-07T17_13_53_907Z-debug.log
gmacario@gmpowerhorse:~/github/aquariophilie/blobfishes/client (main)*$
```

**Comment by gmacario** [comment]
I found this relevant thread: 
Reference: https://githubmemory.com/repo/sveltejs/kit/issues/2646

**Comment by gmacario** [comment]
Fix via #129
